packages/cashier/src/components/cashier-locked/cashier-locked.tsx

```diff
@@ -1,35 +1,23 @@
 import React from 'react';
-import { Authorize, GetAccountStatus } from '@deriv/api-types';
-import { connect } from 'Stores/connect';
 import EmptyState from 'Components/empty-state';
 import getMessage from './cashier-locked-provider';
-import { TRootStore } from 'Types';
+import { useStore } from '../../hooks';
```

Comment:
Can't use an alias for this?

Reply:
@farzin-fs 😊 the same thing happened with me @hamid-deriv 😊